### PR TITLE
Default interval for watchd

### PR DESCRIPTION
**Pull request by pmbohm**

### CODE REVIEW

File: cookbooks/imos_metrics/templates/default/metrics.conf.erb [hunk 1]
```diff
@@ -5,6 +5,7 @@ TypesDB "/usr/share/collectd/types.db" "/etc/collectd/collectd.conf.d/types.db"
 <Plugin python>
     LogTraces true
     Interactive false
+    Interval <%= node['imos_metrics']['sumologic']['interval'] %>
```
Comment (jonescc) [inline cookbooks/imos_metrics/templates/default/metrics.conf.erb]:
```
Sep 12 14:37:02 vagrant-2-aws-syd collectd[940]: python plugin: Ignoring unknown config key "Interval".
```
Should be before <Plugin python> to set it for all metrics collected.

**Comment by pmbohm** [comment]
Tested in vagrant
Seeing logs now such as:
` sudo less /var/log/syslog | grep collectd | more`

```
Sep 13 05:15:58 vagrant collectd[7132]: Sent https request with batch_size=16 got response_code=200
Sep 13 05:16:59 vagrant collectd[7132]: Sent https request with batch_size=16 got response_code=429
Sep 13 05:17:59 vagrant collectd[7132]: Sent https request with batch_size=16 got response_code=429
Sep 13 05:22:59 vagrant collectd[7132]: message repeated 5 times: [ Sent https request with batch_size=16 got response_code=429]
Sep 13 05:23:59 vagrant collectd[7132]: Sent https request with batch_size=16 got response_code=429
Sep 13 05:24:59 vagrant collectd[7132]: Sent https request with batch_size=16 got response_code=429
Sep 13 05:25:59 vagrant collectd[7132]: Sent https request with batch_size=16 got response_code=429
Sep 13 05:26:59 vagrant collectd[7132]: Sent https request with batch_size=16 got response_code=429
Sep 13 05:31:59 vagrant collectd[7132]: message repeated 5 times: [ Sent https request with batch_size=16 got response_code=429]
Sep 13 05:32:59 vagrant collectd[7132]: Sent https request with batch_size=16 got response_code=429
Sep 13 05:33:59 vagrant collectd[7132]: Sent https request with batch_size=16 got response_code=429
Sep 13 05:34:59 vagrant collectd[7132]: Sent https request with batch_size=16 got response_code=429
Sep 13 05:35:59 vagrant collectd[7132]: Sent https request with batch_size=16 got response_code=429
Sep 13 05:40:59 vagrant collectd[7132]: message repeated 5 times: [ Sent https request with batch_size=16 got response_code=429]
```

**Comment by pmbohm** [comment]
@jonescc It seems that this install of collectd does not have `Interval` in its base conf file, unlike https://github.com/aodn/cloud-deploy/pull/177

```
vagrant@vagrant-2-nec-hob:~$ grep -o '^[^#]*' /etc/collectd/collectd.conf
FQDNLookup true
LoadPlugin syslog
<Plugin syslog>
	LogLevel info
</Plugin>
LoadPlugin battery
LoadPlugin cpu
LoadPlugin df
LoadPlugin disk
LoadPlugin entropy
LoadPlugin interface
LoadPlugin irq
LoadPlugin load
LoadPlugin memory
LoadPlugin processes
LoadPlugin rrdtool
LoadPlugin swap
LoadPlugin users
<Plugin df>


	FSType rootfs

	FSType sysfs
	FSType proc
	FSType devtmpfs
	FSType devpts
	FSType tmpfs
	FSType fusectl
	FSType cgroup
	IgnoreSelected true
</Plugin>
<Plugin rrdtool>
	DataDir "/var/lib/collectd/rrd"
</Plugin>
<Include "/etc/collectd/collectd.conf.d">
	Filter "*.conf"
</Include>
```

**Comment by jonescc** [comment]
That's because its the default collectd config.  Cloud-deploy is installing its own (which is not recommended).   Normally you use the collectd.conf.d directory to customise the install as collectd should be left to manage the core config.

**Pull request merged**